source/_components/image_processing.facebox.markdown

```diff
@@ -24,6 +24,7 @@ MB_KEY="INSERT-YOUR-KEY-HERE"
 
 sudo docker run --name=facebox --restart=always -p 8080:8080 -e "MB_KEY=$MB_KEY"  machinebox/facebox
 ```
+If you are on an unsecured network you can run Facebox with a username and password, add ` -e "MB_BASICAUTH_USER=my_username" -e "MB_BASICAUTH_PASS=my_password" `
```

Comment:
The requests are still not encrypted right? So we shouldn't recommend this on an unsecured network?

Reply:
I hadn't considered that, but you are correct. Whats the standard statement?

Reply:
I guess the statement should be: 'You can run Facebox .. but bare in mind that the component does not encrypt these credentials'?